### PR TITLE
Add OntologyMappingProcessor class

### PR DESCRIPTION
**Pull request by svenvanderburg**
Add OntologyMappingProcessor class for filtering fetched ontology mappings and converting them to RDF triples.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -11,7 +11,6 @@ source = zib_uploader
 
 [tool:pytest]
 testpaths = tests
-addopts = --cov --cov-report xml --cov-report term --cov-report html
```
Comment (svenvanderburg) [inline setup.cfg:14]: I removed this because it prevented me to run any tests. Not sure why we need it?

Comment (dsmits) [reply setup.cfg:14]: I believe it's from the escience cookiecutter template but it's fine to remove it. For local coverage analysis I like to use pycharms built-in coverage anyway. I noticed that having coverage installed locally prevents me from using the debugger.

File: zib_uploader/zib_uploader.py [hunk 2]
```diff
@@ -1,9 +1,78 @@
...
+
+from zib_uploader.fetch_data import OntologyMapping
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_MAPPING_PREDICATE = 'http://www.w3.org/2004/02/skos/core#closeMatch'
+DEFAULT_VALID_SOURCES = ('CUI',)
+
+
+class OntologyMappingsProcessor:
```
Comment (dsmits) [inline zib_uploader/zib_uploader.py]: Can you rename the module to `ontology_mappings_processor`? The `zib_uploader` module is another remnant of the escience project template.

**Review by dsmits — CHANGES_REQUESTED** [review]
Could you rename the zib_uploader module? It might also be nice if you add a script in the root project dir that actually runs fetcher and processor.

File: setup.cfg (hunk 1 above)
Comment (svenvanderburg) [reply setup.cfg:14]: Ok, cool

File: zib_uploader/zib_uploader.py (hunk 2 above)
Comment (svenvanderburg) [reply zib_uploader/zib_uploader.py]: I have renamed it to `process_data.py`, so it is a bit more general.

Comment (svenvanderburg) [reply zib_uploader/zib_uploader.py]: I have added a script in `scripts/get_ontology_mappings`. Can you check to see if you like it?

**Review by dsmits — APPROVED** [review]
Toppie

**Pull request merged**